### ISSUE TITLE
Additional maintainers wanted

### ISSUE THREAD

**Issue by rhansen** [ddclient/ddclient#773]
Are you interested in helping maintain ddclient?  We're looking for additional maintainers to help review PRs, triage issues, fix bugs, etc.  Please comment here if interested, or contact @rhansen directly.

**Comment by indrajitr** [comment]
@rhansen thank you for opening up the offer. Will be happy to be of help.

**Comment by rhansen** [comment]
Oops, looks like I don't have permissions to add maintainers.  @wimpunk @SuperSandro2000 or @rrthomas, would you please bump my privileges?  TIA!

**Comment by rrthomas** [comment]
Did someone already do that @rhansen? You seem to be an admin already…

**Comment by rrthomas** [comment]
OK, I now made you a team maintainer of the members team, hopefully that does the right thing.

**Comment by SuperSandro2000** [comment]
You need to be owner to invite new members, otherwise you can probably just change permissions of existing members.

**Comment by rhansen** [comment]
Thanks for volunteering @indrajitr. I sent an invite.

And thanks @rrthomas for responding so quickly.

I'm going to leave issue open a little while longer in case anyone else would like to volunteer.

**Comment by indrajitr** [comment]
@rhansen, @rrthomas, @SuperSandro2000 - can you add @beadon as a member, please? He has volunteered to be a maintainer. Or bump up my privilege so that I can take care. Thanks!

**Comment by rrthomas** [comment]
Done!

**Comment by indrajitr** [comment]
@rrthomas, Thanks! But I don't see the option to add another member (it says my request is pending). Neither do I see @beadon as a member.

**Comment by rrthomas** [comment]
I invited @beadon to the members team. I assumed that's what you meant?

**Comment by indrajitr** [comment]
Awesome! He is not listed as member, probably because he is yet to accept the invitation.
Possible to bump up my privilege as well so that I don't have to bother you guys?

**Comment by rrthomas** [comment]
@indrajitr you're already in the maintainers group, which gives you Admin privileges, which according to the GitHub summary for that role allows you to add collaborators to the project. So I'm not sure why you can't do this (or indeed, where your request to do it would be listed!).

**Comment by rrthomas** [comment]
> Awesome! He is not listed as member, probably because he is yet to accept the invitation.

Yes, that's my understanding. I checked that the invitation has been issued.

**Comment by indrajitr** [comment]
> [@indrajitr](https://github.com/indrajitr) you're already in the maintainers group, which gives you Admin privileges, which according to the GitHub summary for that role allows you to add collaborators to the project. So I'm not sure why you can't do this (or indeed, where your request to do it would be listed!).

Yep, I can add outside collaborators per repo (which is probably different from org members). Adding an [org member needs the owner privilege](https://docs.github.com/en/organizations/organizing-members-into-teams/adding-organization-members-to-a-team). But I think we are good at the moment.

Thanks for all the help!

Edit: I see what might have happened. Maintainer role ≠ Member in a team named 'maintainer'. Per [the docs](https://docs.github.com/en/organizations/organizing-members-into-teams/assigning-the-team-maintainer-role-to-a-team-member), the `maintainer` role is necessary to have the ability to add a member.

**Comment by rrthomas** [comment]
Aha. I don't think I set up the teams, but it's confusing. I don't think I should start fiddling with this. If it becomes a problem, someone more active should sort it out!

**Comment by indrajitr** [comment]
Yes, let's not worry about it right now. 

Anybody with the `owner` role should be able to sort it out. You already have the `owner` role, so I can always reach out to you if something comes up. 

And if this keeps becoming a recurring bottleneck, one of the members with the `owner` role can bump up the privilege of whoever is/are active maintainer(s) then.

**Comment by beadon** [comment]
Just a fyi - I have not seen an invite yet.

**Comment by rrthomas** [comment]
@beadon I've just resent the invite.

**Comment by beadon** [comment]
> @beadon I've just resent the invite.

So weird.  No email or a notification of any kind on GitHub.

**Comment by beadon** [comment]
Buried in my email.  Finally found it deep ....
